WordPress-iOS-Shared/Core/WPStyleGuide.h

```diff
@@ -27,48 +27,65 @@
 + (UIFont *)tableviewSubtitleFont;
 + (UIFont *)tableviewSectionHeaderFont;
 
-// Colors
+// Color
 + (UIColor *)wordPressBlue;
-+ (UIColor *)baseLighterBlue;
-+ (UIColor *)baseDarkerBlue;
 + (UIColor *)lightBlue;
-+ (UIColor *)newKidOnTheBlockBlue;
-+ (UIColor *)midnightBlue;
++ (UIColor *)mediumBlue;
++ (UIColor *)darkBlue;
++ (UIColor *)gray;
++ (UIColor *)lightGrey;
```

Comment:
Oh good catch! Thought I'd kept them consistent.  For fun, http://www.greyorgray.com/